extensions/big-data-cluster/src/bigDataCluster/dialog/bdcDashboardModel.ts

```diff
@@ -23,10 +23,14 @@ export class BdcDashboardModel {
 	private _endpointsLastUpdated: Date;
 	private readonly _onDidUpdateEndpoints = new vscode.EventEmitter<EndpointModel[]>();
 	private readonly _onDidUpdateBdcStatus = new vscode.EventEmitter<BdcStatusModel>();
-	private readonly _onError = new vscode.EventEmitter<Error>();
+	private readonly _onBdcStatusError = new vscode.EventEmitter<Error>();
```

Comment:
This is so many events. Couldn't we just refactor to keep 1 event, but add data (e.g. an enum with the type of error)? That seems like a better design

Reply:
Eh, sure. I didn't think it was worth refactoring for just 1 extra event but yeah it's slightly cleaner to do it that way.